jenkins-integration/beaker/gatling/10_start_gatling_scenario.rb

```diff
@@ -0,0 +1,35 @@
+test_name "Start gatling scenario"
+
+simulation_id = ENV['GATLING_SIMULATION_ID']
+gatling_scenario = ENV['GATLING_SCENARIO']
+sut_hostname = ENV['GATLING_SUT_HOSTNAME']
```

Comment:
It looks like you can. As simple as `sut_hostname = master.hostname` If you've set up a node with a `master` role. I'll switch to that